### ISSUE TITLE
Convert Jira formatted descriptions on migration

### ISSUE THREAD

**Issue by AmitNaveh23** [solidify/jira-azuredevops-migrator#135]
there was an issue about it already that was closed but it still doesn't work as expected for me.
first, it maps the description to both "Repro steps" and "Description".
second, the formatting works only in what was mapped to "Repro steps" even though it has the same value as in "description".

![image](https://user-images.githubusercontent.com/54633588/69725803-b1a50480-1127-11ea-9fab-df58e109fe2c.png)

**Comment by MOlausson** [comment]
@AmitNaveh23 which version are you using?

**Comment by AmitNaveh23** [comment]
@MOlausson  latest version currently : 2.2.59

**Comment by madkoo** [comment]
@AmitNaveh23 we have released a new version that should have fixed this issue you have reported. With the new release, we have introduced a new mapping "MapRendred". If you want the rendered output of description or comment field to be migrated you will need to add a new mapping in your config file like so:  
`
 {
        "source": "description",
        "target": "System.Description",
        "mapper":"MapRendered"
 }
`

I will close this issue, but if you still have problems feel free to reach out.
